src/styles/blocks/_variables.scss

```diff
@@ -0,0 +1,5 @@
+html {
```

Comment:
Variables should be defined in a Sass variables file, instead of using CSS variables inside the html selector.